contents/handbook/company/team/lottie-coxon.md

```diff
@@ -0,0 +1,80 @@
+
+I am apart of the Design team, specialising in visual flare, AKA branding and hedgehogs. Basically, I take all the character of our company and the team and push it into the visuals. PostHog is my first official 'job' job, but during my time at university I created my own bespoke t-shirt painting business and worked as a brand designer for three Oxford Balls. 
+
+I love good conversation, anything honey roasted and bright blue skies. My guilty pleasure is fashion but obviously I veto any fast fashion vendors - sustainable all the way, people! Oh, and I love to create things, and usually always have a small side project running at all times, such as making my own jewellery, painting flower pots or hand making thank you cards. I'm convinced I have no left brain, just one massive right one.  
+
+
+## Areas of responsibility
+
+- Anything to do with hedgehog mascots - I'm all over it.
+- Custom blog artwork
+- Some marketing projects and branding
+- Sometimes I get involved with layouts but usually that is up to Cory
+- Team images and occasionally organising team bios
+- Merchandise 
+- Basically anything that requires artwork I'm ya (wo)man 
+
+## Quirks
+
+- I always try to make others feel as comfortable as possible.
+
+- Sometimes I listen to a song over and over because it sounds so darn good.
+
+- I love a good bit of control in all areas of my life, which obviously includes work. This means I like to know what people want of me, and not knowing what people expect from me makes me anxious and fidgety (the worst kind of Lottie). This also means I will message you if I need to know something even if it seems silly or small.
+
+- While surprises are fun for some, they aren't for me. Please give me heads up on upcoming projects that will need my attention. 
+
+- For some reason I always think I am in trouble - so please start emails or messages with the subject of the message so I can stop worrying. 
+
+- I love telling people exciting things, and I am a massive blabber mouth. Therefore I will talk about the secret Christmas Party plans or secret Offsite locations if I am trusted with them. So don't trust me, please.
+
+- I am unashamedly open and will tell you if I'm stressed, sad, happy or confused - and even embarrassing stories if you are lucky.
+
+- I have a tendency to be a bit of a pushover and a people pleaser - I am working on it, but please be considerate when asking me to do something for you as I will try to bend over backwards to help someone in need. But if it is a real design emergency know that I will always be there to help.
+
+- I am not a perfectionist, but if something doesnt feel right I won't drop it until I am happy with the outcome. Sometimes this is jarring for some as I won't stop stirring the pot, but I trust my gut instinct, and my gut has a good track record. 
+
+- I love a good goofy joke so please humour me with your with and charm - it really brightens my day. 
```

Comment:
```suggestion
- I love a good goofy joke so please humour me with your wit and charm - it really brightens my day. 
```

Reply:
Right, spelling bot is fixed. This change should stick this time. If it doesn't, may be quickest just to use a synonym.